### DISCUSSION TITLE
Support Mud Client Media Protocol (MCMP)

### DISCUSSION THREAD

**Discussion by mpconley** [Blightmud/Blightmud#281] [Ideas]
A good backlog item :)

If you have mastered Mud Sound Protocol, consider updating to Mud Client Media Protocol as a next step!  Find out the details on the specification here: https://wiki.mudlet.org/w/Standards:MUD_Client_Media_Protocol

This is part of Mudlet and BeipMu clients so far.  Grapevine is working on it.

**Comment by LiquidityC** [comment]
There's one of the guys who's already built support for this in his script. Since it's based on GMCP it could easily be built in pure lua. This is definitely a good plugin functionality. Perhaps if we just add some lua bindings to play sound.

**Comment by LiquidityC** [comment]
Audio playback is in the works even if it's currently slightly unstable. However things pan out this is something that should be implemented as a plugin and not added Blightmud core. I will leave this open for reference but once audio is stable and released I will close this issue.
